pkgs/tools/misc/most/default.nix

```diff
@@ -9,6 +9,11 @@ stdenv.mkDerivation rec {
     sha256 = "008537ns659pw2aag15imwjrxj73j26aqq90h285is6kz8gmv06v";
   };
 
+  patches = [
+    # Upstream patch to fix parallel build failure
+    ./parallel-make.patch
```

Comment:
Is this upstreamed? Can we fetch from there?

Reply:
Yes, it's an upstreamed patch. From what I understand there is no HTTP UI for most project. Only git (and snapshots): https://www.jedsoft.org/snapshots/.